docs/modules/ROOT/pages/documentation/microprofile/config/directory.adoc

```diff
@@ -101,34 +107,62 @@ Usually this config source is used to map secrets mounted by Kubernetes or
 Docker to a directory inside a running container.
 
 Once you configured a directory to read a config (secrets) from, you need to
-make sure properties in your codebase map to files.
+make sure file names correspond to properties in your codebase.
 
 === Map property names to flat file hierarchy
 
 _Since *5.183*&nbsp;_
 
 1. Say you have two properties `property1` and `foo.bar.property2`.
-2. Payara is configured with secret directory `/home/payara/.secrets`
-3. Secrets mounted as files to `/home/payara/.secrets/property1` and
-   `/home/payara/.secrets/foo.bar.property2` will be read.
+2. Payara is configured with secret directory `/home/payara/secrets`
+3. Secrets mounted as files to `/home/payara/secrets/property1` and
+   `/home/payara/secrets/foo.bar.property2` will be read.
 
-Remember that the files may only contain the secret, no newline, metadata, etc.
+Remember that the files may only contain the secret or other configuration
+value - no newlines, metadata, etc! Plaintext files only, no file extensions
+allowed.
 
 === Map property names to a sub-directory structure
 
-_Since *5.2020.7*&nbsp;_
+_Since *5.2021.1*&nbsp;_
```

Comment:
Please change this if necessary. Can be done via GH suggestion comment for example.